pages/index.vue

```diff
@@ -2,18 +2,26 @@
   <div class="main">
     <h1 class="title">I Cannot Design.</h1>
     <hr>
+    <marquee direction="down" width="250" height="200" behavior="alternate">
+      <marquee behavior="alternate">
+        Under construction!!!1!1one
+      </marquee>
+    </marquee>
     <img src="/construction.gif"
          height="36"
          width="343"
          alt="This site is under eternal construction.">
+    <img src="/underconstruction.gif"
+         height="180"
+         width="158"
+         alt="Still under construction">
     <div>
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
       <img src="/anim0206-1_e0.gif" alt="A cartoon construction worker eating alternating between eating a sandwich and drinking water.">
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
       <img src="/anim0206-1_e0.gif" alt="A cartoon construction worker eating alternating between eating a sandwich and drinking water.">
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
     </div>
-
```

Comment:
please bring back the whitespace

Reply:
fixed in https://github.com/fimion/icannotdesign/pull/1/commits/c7ac4049b27de8ed47c7219ffc2d058c5da14b33